control/cli.py

```diff
@@ -394,9 +403,93 @@ def gw_version(self, args):
 
         return gw_info.status
 
+    def gw_get_log_level(self, args):
+        """Get gateway's log level"""
+
+        out_func, err_func = self.get_output_functions(args)
+        req = pb2.get_gateway_log_level_req()
+        try:
+            ret = self.stub.get_gateway_log_level(req)
+        except Exception as ex:
+            ret = pb2.req_status(status = errno.EINVAL, error_message = f"Failure getting gateway log level:\n{ex}")
+
+        if args.format == "text" or args.format == "plain":
+            if ret.status == 0:
+                level = GatewayEnumUtils.get_key_from_value(pb2.GwLogLevel, ret.log_level)
+                out_func(f"Gateway log level is \"{level}\"")
+            else:
+                err_func(f"{ret.error_message}")
+        elif args.format == "json" or args.format == "yaml":
+            out_log_level = json_format.MessageToJson(ret,
+                                                indent=4,
+                                                including_default_value_fields=True,
+                                                preserving_proto_field_name=True)
+            if args.format == "json":
+                out_func(f"{out_log_level}")
+            elif args.format == "yaml":
+                obj = json.loads(out_log_level)
+                out_func(yaml.dump(obj))
+        elif args.format == "python":
+            return ret
+        else:
+            assert False
+
+        return ret.status
+
+    def gw_set_log_level(self, args):
+        """Set gateway's log level"""
+
+        out_func, err_func = self.get_output_functions(args)
+        log_level = None
+
+        if args.level:
+            log_level = args.level.lower()
+
+        try:
+            req = pb2.set_gateway_log_level_req(log_level=log_level)
+        except ValueError as err:
+            errstr = str(err)
+            if errstr.startswith("unknown enum label "):
+                errstr = errstr.removeprefix("unknown enum label ")
+                self.cli.parser.error(f"invalid log level {errstr}")
+            else:
+                self.cli.parser.error(f"{err}")
```

Comment:
Could it be be simplified? 
```suggestion
            self.cli.parser.error(f"invalid log level {log_level}, error {err}")
```

Reply:
I prefer to have clearer messages in case of an error to saving a few lines in the code. But fine, I will change that.